app/uk/gov/hmrc/traderservices/wiring/AppConfig.scala

```diff
@@ -107,6 +111,10 @@ class AppConfigImpl @Inject() (config: ServicesConfig) extends AppConfig {
   override val signOutUrl: String = config.getString("urls.signOut")
   override val researchBannerUrl: String = config.getString("urls.researchBanner")
 
+  override val timeout: Int = config.getInt("session._timeoutSeconds")
```

Comment:
why underscore?

Reply:
I copied it from another hmrc project so I thought it was there for a reason